src/integer_mod_q/mat_polynomial_ring_zq/default.rs

```diff
@@ -103,3 +134,106 @@ mod test_new {
         let _ = MatPolynomialRingZq::new(2, 2, &modulus);
     }
 }
+
+#[cfg(test)]
+mod test_identity {
+    use crate::{
+        integer::PolyOverZ,
+        integer_mod_q::{MatPolynomialRingZq, ModulusPolynomialRingZq},
+        traits::GetEntry,
+    };
+    use std::str::FromStr;
+
+    /// Tests if an identity matrix is set from a zero matrix.
+    #[test]
+    fn identity() {
+        let modulus = ModulusPolynomialRingZq::from_str("3  1 0 1 mod 17").unwrap();
+        let matrix = MatPolynomialRingZq::identity(10, 10, &modulus);
+
+        for i in 0..10 {
+            for j in 0..10 {
```

Comment:
```suggestion
        for i in 0..matrix.get_num_rows() {
            for j in 0..matrix.get_num_columns() {
```
Optional. Just to make the test a bit more robust for change of matrix dimensions.